### ISSUE TITLE
Equivalence of stage-3 fu'ivla with different rafsi forms

### ISSUE THREAD

**Issue by Wuzzy2** [lojban/cll#446]
The CLL states that 2 lujvo are considered equivalent if all their rafsi refer to the same “parent words”.

However, the CLL doesn't say anything similar about stage-3 fu'ivla. This is a question I'd like to see clarified in the CLL:

> Are two stage-3 fu'ivla that only differ in their rafsi that both refer to the same “parent word” considered be equivalent in meaning (similar to lujvo equivalence)? 
> 
> E.g. is “gism,r,bla” considered to be equivalent to “gim,r,bla” in meaning, or are those two completely separate words with (possibly) different meanings?

**Comment by phma** [comment]
I think they should be considered alternative forms of the same word, just as "xauzma" and "xagmau" are alternative forms of the same word.

**Comment by Wuzzy2** [comment]
I agree.

**Comment by vpbroman** [comment]
Defining these equivalences, do we then have to impose restrictions on stage-4 fuhivla that they must not look like a stage-3 fuhivla?
If stage-4 forms may collide with stage-3 forms, we have to know the etymology of a fuhivla to be sure what forms might be equivalent to it and how to look it up in a lexicon.

**Comment by Wuzzy2** [comment]
I don't think so. A stage-3 fu'ivla is clearly defined to be of the form `RAFSI + HYPHEN + REST`.

Like `cid,r,spageti`. The fu'ivla stage-3 is, as far I understood, only defined on a grammatical level. So if the fu'ivla matches this syntax, it is stage-3, otherwise it is not.

The notion of a "stage-4 fu'ivla that looks like a stage-3 fu'ivla" doesn't make sense to me.

So no. No restrictions on stage-4 fu'ivla need to be applied, it wouldn't make sense.

----

Anyway, this is besides the point.

I noticed something. Currently, the CLL seems to state that two stage-3 fu'ivla with different rafsi forms (but for the same word) are "distinct", as suggeted here, in the fu'ivla chapter:

> Technically, “ricrxaceru” and “tricrxaceru” are distinct fu'ivla, but they would surely be given the same meanings if both happened to be in use. 

It is a bit unfortunate that the CLL didn't add an equivalence rule like for lujvo. It might be too late to change that now …
